libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx

```diff
@@ -2,51 +2,47 @@ import type { RootState } from '../../core/state/templates/store';
 import { useSelector } from 'react-redux';
 import { TemplateCard } from './cards/templateCard';
 import { TemplatePanel } from '../panel/templatePanel/templatePanel';
-// import type { Template, LogicAppsV2 } from '@microsoft/logic-apps-shared';
+import type { Template, LogicAppsV2 } from '@microsoft/logic-apps-shared';
 
-export const TemplatesDesigner = (
-  //   {
-  //   createWorkflowCall,
-  // }: {
-  //   createWorkflowCall: (
-  //     workflowName: string,
-  //     workflowKind: string,
-  //     workflow: LogicAppsV2.WorkflowDefinition,
-  //     connectionsData: any,
-  //     parametersData: Record<string, Template.ParameterDefinition>
-  //   ) => Promise<void>;
-  // }
-) => {
-  // const { workflowName: existingWorkflowName } = useSelector((state: RootState) => state.workflow);
-  // const { workflowName, kind, workflowDefinition, parameters } = useSelector((state: RootState) => state.template);
+export const TemplatesDesigner = ({
+  createWorkflowCall,
+}: {
+  createWorkflowCall: (
+    workflowName: string,
+    workflowKind: string,
+    workflow: LogicAppsV2.WorkflowDefinition,
+    connectionsData: any,
```

Comment:
NIT: Can we put a data type to connectionsData?

Reply:
That will be in another PR addressing connections part, which is incomplete